ngx-tools/src/README.md

```diff
@@ -25,6 +25,7 @@ isArray([]); // Returns: true
 - [`noop`](#noop)
 - [`retryWithBackoff`](#retrywithbackoff)
 - [`returnValuesByKeys`](#returnvaluesbykeys)
+- [`setFormControlValue`](#setFormControlValue)
```

Comment:
TOC needs to be generated by running `yarn run docs`. This link will break on GitHub.

Reply:
Hmm... thought I did the same thing last time..  so I should leave TOC untouched and update the content, then run `yarn run docs`?

Reply:
Hmm.. possibly I missed it? 

Yep. Notice the TOC is wrapped by HTML comments that say not to edit the TOC. There is a tool in out docs generation process that will keep all TOCs updated according to the content automatically.